core/src/test/java/greencity/controller/EventsControllerTest.java

```diff
@@ -67,4 +69,76 @@ void getEventsCreatedByUser() {
 
     }
 
+    @Test
+    @SneakyThrows
+    void getUserEventsTest() {
+        int pageNumber = 0;
+        int pageSize = 20;
+
+        Pageable pageable = PageRequest.of(pageNumber, pageSize);
+        mockMvc.perform(get(EVENTS_CONTROLLER_LINK + "/myEvents").principal(principal))
+            .andExpect(status().isOk());
+
+        verify(eventService, times(1))
```

Comment:
`times(1)` could be removed
  `verify(eventService)` equals to   `verify(eventService, times(1))`